saskatoon/harvest/filters.py

```diff
@@ -50,51 +61,82 @@ class HarvestFilter(filters.FilterSet):
         required=False
     )
 
+
+class PropertyFilter(filters.FilterSet):
+
     class Meta:
-        model = Harvest
-        fields = (
-            'status',
-            'pick_leader',
+        model = Property
+        fields = [
+            'is_active',
+            'authorized',
+            'neighborhood',
             'trees',
-            'property__neighborhood',
-            'season',
-        )
+            'ladder_available',
+            'ladder_available_for_outside_picks',
+            'pending',
+        ]
 
+    is_active = filters.BooleanFilter(
+        label=_("Active"),
+        help_text=""
+    )
+
+    authorized = filters.ChoiceFilter(
+        choices=[
+            (2, _("Not yet authorized")),
+            (1, _("Authorized")),
+            (0, _("Unauthorized"))
+        ],
+        label=_("Authorized"),
+        help_text="",
+        method='authorized_filter'
+    )
 
-class PropertyFilter(filters.FilterSet):
-    is_active = filters.BooleanFilter(help_text="")
-    authorized = filters.BooleanFilter(help_text="")
     neighborhood = filters.ModelChoiceFilter(
         queryset=Neighborhood.objects.all(),
         label=_("Neighborhood"),
         help_text="",
         required=False
     )
+
     trees = filters.ModelChoiceFilter(
         queryset=TreeType.objects.all(),
         label=_("Tree"),
         help_text="",
         required=False
     )
-    ladder_available = filters.BooleanFilter(help_text="")
-    ladder_available_for_outside_picks = filters.BooleanFilter(help_text="")
-    pending = filters.BooleanFilter(help_text="", label=_("Pending validation"))
 
-    class Meta:
-        model = Property
-        fields = [
-            'is_active',
-            'authorized',
-            'neighborhood',
-            'trees',
-            'ladder_available',
-            'ladder_available_for_outside_picks',
-            'pending',
-            ]
+    ladder_available = filters.BooleanFilter(
+        help_text=""
+    )
+
+    ladder_available_for_outside_picks = filters.BooleanFilter(
+        help_text=""
+    )
+
+    pending = filters.BooleanFilter(
+        help_text="",
+        label=_("Pending validation")
+    )
+
+    def authorized_filter(self, queryset, name, choice):
+        if choice == '2':
+            return queryset.filter(authorized__isnull=True)
+        return queryset.filter(authorized=bool(int(choice)))
```

Comment:
Would it make send to wrap this logic inside a custom `BooleanOptionalFilter` class? Maybe there are other optional boolean fields that would benefit from using it ?

This is a minor comment and does not have to be implemented in this PR.